docs/src/main/asciidoc/status-include.adoc

```diff
@@ -0,0 +1,20 @@
+[NOTE]
+====
+This technology is considered {extension-status}.
+
+ifeval::["{extension-status}" == "experimental"]
+In _experimental_ mode, early feedback is requested to mature the idea.
+There is no guarantee of stability nor long term presense in the platform until the solution matures.
+Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing list] or as issues in our https://github.com/quarkusio/quarkus/issues[GitHub issue tracker].
+endif::[]
+ifeval::["{extension-status}" == "preview"]
+In _preview_, backward compatibility and presence in the ecosystem is not guaranteed.
+Specific improvements might require to change configuration or APIs and plans to become _stable_ are under way.
+Feedback is welcome on our https://groups.google.com/d/forum/quarkus-dev[mailing list] or as issues in our https://github.com/quarkusio/quarkus/issues[GitHub issue tracker].
+endif::[]
+ifeval::["{extension-status}" == "stable"]
+Being _stable_, backward compatibility and presence in the ecosystem are taken very seriously.
+endif::[]
+
+For a full list of possible extension statuses, check our https://some.url/faq#status[FAQ entry].
```

Comment:
I fixed the URL here to point to https://quarkus.io/faq/#extension-status but we will need to add the content there.

Reply:
FAQ created